frontend/src/containers/SelectedCourses/index.js

```diff
@@ -109,10 +135,11 @@ function SelectedCourses({ history }) {
           Tambah Agenda
         </Button>
         <Button
-          onClick={saveSchedule}
-          disabled={isConflict || totalCredits > 24 || schedules.length === 0}
+          onClick={() => !isEditing ? saveSchedule() : totalCredits === 0 ? handleDeleteSchedule() : updateSchedule()}
```

Comment:
Please use `schedules.length === 0` rahter than `totalCredits === 0` like the one in line 142 so it correctly update the schedule when have no course but have agendas.

Reply:
Noted. Thanks for pointing out.

Reply:
Resolved @fairuzi10 .